### PR TITLE
Add MQ issue template

### PR DESCRIPTION
**Pull request by pallavit**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/05_mqissue.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: MQ Improvements Tracker
```
Comment (jsquire) [inline .github/ISSUE_TEMPLATE/05_mqissue.yml:1]: If we don't want customers using this... we probably should consider whether we want to add this.

Comment (pallavit) [reply .github/ISSUE_TEMPLATE/05_mqissue.yml:1]: <Had an offline conversation with @jsquire> We will add this new template and  use the wait and watch policy for testing.

Comment (pallavit) [reply .github/ISSUE_TEMPLATE/05_mqissue.yml:1]: @jsquire let me know if you have any concerns.

Comment (jsquire) [reply .github/ISSUE_TEMPLATE/05_mqissue.yml:1]: :shipit:

Comment (pallavit) [reply .github/ISSUE_TEMPLATE/05_mqissue.yml:1]: Awesome, please approve :)

File: .github/ISSUE_TEMPLATE/05_mqissue.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+description: Use this template to track improvements.
+title: "[MQ][Issue]"
+labels: ["MQ"]
```
Comment (jsquire) [inline .github/ISSUE_TEMPLATE/05_mqissue.yml:4]: Can we also add "Client" here?

**Review by jsquire — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/05_mqissue.yml (hunk 2 above)
Comment (pallavit) [reply .github/ISSUE_TEMPLATE/05_mqissue.yml]: Done

**Pull request merged**